ark/utils/data_utils_test.py

```diff
@@ -134,3 +138,54 @@ def test_split_img_stack():
 
         assert np.array_equal(sample_chan_1, data_xr[0, :, :, 0].values)
         assert np.array_equal(sample_chan_2, data_xr[0, :, :, 1].values)
+
+
+def test_relabel_img_array():
+    x = y = 5
+    img_arr = np.arange(1, x * y + 1).reshape((x, y))
```

Comment:
Great, let's add a case where multiple pixels have the same label to more accurately simulate. You can np.repeat to modify the above code, or make a new test

Reply:
I added another test